webapp/channels/src/components/custom_status/date_time_input.tsx

```diff
@@ -145,7 +145,7 @@ const DateTimeInputContainer: React.FC<Props> = (props: Props) => {
     const datePickerProps: DayPickerProps = {
         initialFocus: isPopperOpen,
         mode: 'single',
-        selected: currentTime,
+        selected: time.toDate(),
```

Comment:
If you select a date that is not part of the current month, the calendar will open in the current month still. You can fix this by adding `defaultMonth: time.toDate(),`